Observatory-Control-System/EAS/DIMM/ops.rst

```diff
@@ -106,8 +106,24 @@ and within the OpenTPL:
         AUTH OK 0 0
         1 set ameba.mode=1
         
+It is common to see ``EVENT WARN`` messages regarding the state of ameba. This does not necessarily indicate an issue.
 
-Verify the DIMM CSC is enabled and data is appearing in the EFD.
+The status of ameba can be checked with with following command to *tt-master*. This can be useful to check if the DIMM will operate tonight, or if there were concerning warning messages when starting ameba.
+
+.. code-block:: bash
+
+        AUTH OK 00
+        1 get ameba.mode
+
+Which will reply something like the following with 1 or 0 to indicate on or off.
```

Comment:
I think here it wouldn't harm to more specifically declare ``ameba.mode=1`` is on, and 0 means it's off.